.github/workflows/build-deps.yml

```diff
@@ -0,0 +1,45 @@
+name: Build dependencies
+
+on:
+  pull_request:
+    # paths:
+    # - datadog_checks_dev/datadog_checks/base/data/agent_requirements.in
```

Comment:
Will this be enabled later, or kept for troubleshooting? If the latter, a comment to that effect would be nice.